classes/Referrer.php

```diff
@@ -359,16 +359,16 @@ public static function refreshIndex($referrers = null)
      *
      * @param int $idReferrer Referrer ID
      * @param int $idProduct Product ID
-     * @param int $idEmployee Employee ID
+     * @param Employee|null $employee Employee ID
```

Comment:
No fix required for such a small typo :)
```suggestion
     * @param Employee|null $employee Employee
```

Reply:
Fixed